openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -443,6 +443,16 @@ jQuery(function () {
         $('details[open]').not(this).removeAttr('open');
     });
 
+    $('.header-dropdown').on('keydown', function (event) {
+        if (event.key === 'Escape') {
+            $('details[open]').not(this).removeAttr('open');
+        }
+    });
+
+    $('.hamburger-dropdown-menu a').last().on('focusout', function () {
+        $('.hamburger-dropdown-menu a').first().trigger('focus');
+    });
```

Comment:
Stealing from our neighbours at Wikipedia, it seems like the pattern they use is that tabbing through the hamburger menu, when you get to the end, it _exits_. Something like when you `focusout` of the `.hamburger-dropdown-menu`, then close the menu. That seems to be a more standard interaction.

Reply:
Can we also make the other menus behave this way? E.g. the browse menu?

Reply:
Hello Drini, I have updated my code to address both of these comments! All dropdown menus now exit when the user tabs through the end of the menu.